packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -565,4 +587,8 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 	): Promise<FluidObjectNode | undefined> {
 		return this.dataVisualizer?.render(fluidObjectId) ?? undefined;
 	}
+
+	private async editData(fluidObjectId: FluidObjectId, newData: Serializable): Promise<void> {
```

Comment:
We should add a TSDoc comment to this to explain what it does and what the parameters mean. I think there is enough nuance here that a comment would be useful for long-term maintenance.

Reply:
Check new commit